backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java

```diff
@@ -146,9 +153,10 @@ public Cluster build() {
                 new QueryOptions()
                     .setRefreshSchemaIntervalMillis(refreshSchemaIntervalMillis)));
 
-        readTimeoutMillis.map(timeout ->
-            clusterBuilder.withSocketOptions(
-                    new SocketOptions().setReadTimeoutMillis(timeout)));
+        SocketOptions socketOptions = new SocketOptions();
+        readTimeoutMillis.map(socketOptions::setReadTimeoutMillis);
```

Comment:
Here you use map for border effect I thought we agree that is not a good practice with @chibenwa.
Can you use `ifPresent` instead.